src/AzureAuth.Test/CommandMainTest.cs

```diff
@@ -384,6 +388,119 @@ public void TestPromptHintPrefixWithoutOption()
                 .Should().BeEquivalentTo(PromptHintPrefix);
         }
 
+        /// <summary>
+        /// The test to evaluate a normal customized cache file path.
+        /// </summary>
+        [Test]
+        [Platform("Win")] // Only valid on Windows
+        public void TestCacheFileOptionWithNormalFilePath()
+        {
+            CommandMain subject = this.serviceProvider.GetService<CommandMain>();
+            subject.Resource = "f0e8d801-3a50-48fd-b2da-6476d6e832a2";
+            subject.Client = "e19f71ed-3b14-448d-9346-9eff9753646b";
+            subject.Tenant = "9f6227ee-3d14-473e-8bed-1281171ef8c9";
+
+            subject.CacheFilePath = "Z:\\normal.cache";
+            subject.EvaluateOptions().Should().BeTrue();
+            subject.CacheFilePath.Should().Be("Z:\\normal.cache");
+        }
+
+        /// <summary>
+        /// The test to evaluate a relative cache file name in enviroment variables.
+        /// </summary>
+        [Test]
+        [Platform("Win")] // Only valid on Windows
+        public void TestCacheFileOptionWithNormalFilePathFromEnv()
+        {
+            string filenameFromEnv = "C:\\test\\absolute_from_env.cache";
+            this.envMock.Setup(env => env.Get("AZUREAUTH_CACHE_FILE")).Returns(filenameFromEnv);
+
+            CommandMain subject = this.serviceProvider.GetService<CommandMain>();
+            subject.Resource = "f0e8d801-3a50-48fd-b2da-6476d6e832a2";
+            subject.Client = "e19f71ed-3b14-448d-9346-9eff9753646b";
+            subject.Tenant = "9f6227ee-3d14-473e-8bed-1281171ef8c9";
+
+            subject.EvaluateOptions().Should().BeTrue();

```

Comment:
Are we missing an assertion here that the CaceFile property on the `subject` is equal to the expect cache file?

Reply:
😅 I missed that.